group_vars/all

```diff
@@ -53,3 +53,21 @@ ansible_git_repository: git://github.com/buildbot/buildbot-infra
 
 # syslog server to which all syslog messages should be forwarded
 global_syslog_server: 192.168.80.51
+
+# `hosts` declares ip addresses for _all_ machines and jails under Ansible
+# management.
+hosts:
+  ns1:
+    vlan0: "192.168.80.236"
```

Comment:
re host names. non-fqdn names are keys in `hosts`. fqnd names can be produced by concatenating non-fqdn with the domain suffix specific for the interface: I do not envision situations when we might want to have a host name that has to be explicitly specified.